neo/Ledger/TrieReadOnlyStore.cs

```diff
@@ -27,5 +27,9 @@ public UInt256 GetRoot()
             var result = Get(ROOT_KEY);
             return result is null ? UInt256.Zero : new UInt256(result);
         }
+
+        public void Put(byte[] key, byte[] value) { }
```

Comment:
If it's readonly we should throw an exception

Reply:
Will do.

Reply:
done